modules/aleph/src/Aleph/AlephClient.php

```diff
@@ -51,17 +63,19 @@ class AlephClient {
    *   The base url for the Aleph end-point.
    * @param $base_url_rest
    *    The base url for the Aleph REST end-point.
-   *
    * @param $main_library
    *    The main library. For example ICE01.
-   *
    * @param $filter_institution
    *    The institution filter to set. ICE53 for example.
+   * @param $branches
+   *    Filter to certain branches.
    */
-  public function __construct($base_url, $base_url_rest, $main_library, $filter_institution) {
+  public function __construct($base_url, $base_url_rest, $main_library, $filter_institution, $branches) {
     $this->baseUrl = $base_url;
     $this->baseUrlRest = $base_url_rest;
     $this->mainLibrary = $main_library;
+    $this->filterInstitution = $filter_institution;
+    $this->branches = $branches;
```

Comment:
Nitpicking here but is there a reason why `$filter_institution` has the `filter` prefix but `$branches` does not? According to the documentation it is also used for filtering.